index.js

```diff
@@ -221,17 +227,15 @@ class TagInput<T> extends React.PureComponent<Props<T>, State> {
     this.props.onChange(tags);
   }
 
-  scrollToBottom = () => {
-    const y = this.contentHeight - this.scrollViewHeight;
-    if (y <= 0) {
-      return;
-    }
+  scrollToEnd = () => {
     const scrollView = this.scrollView;
     invariant(
       scrollView,
-      "this.scrollView ref should exist before scrollToBottom called",
+      "this.scrollView ref should exist before scrollToEnd called",
     );
-    scrollView.scrollTo({ y, animated: true });
+    setTimeout(() => {
```

Comment:
Why do we need a `setTimeout` here?

Reply:
I know this is ugly, however if it's taken out, both the vertical case (triggered by`onScrollViewContentSizeChange`) and horizontal case (triggered by parent class`onChangeHorizontalText`) are scrolling way beyond the correct position; only the scroll animation triggered by Backspace is correct. 

I guess it because `scrollView.scrollToEnd({ animated: true })` is too fast, and as I tested one of the practical workaround is just to set 0ms timeout.